components/items/ItemsGrid/ItemsGridImage.vue

```diff
@@ -57,6 +73,10 @@ const props = defineProps<{
   variant?: NftCardVariant
 }>()
 
+let { stats } = useCollectionDetails({
```

Comment:
use `const`

Reply:
oh i copied that from, didn't notice.
<img width="656" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/0d0e85b4-355d-4f06-9ab6-0845e71d474b">

I'll change mine to const